crates/cli/src/http/mod.rs

```diff
@@ -160,31 +162,32 @@ async fn get_graphql(
 async fn subscriptions(
     req: HttpRequest,
     stream: web::Payload,
-    schema: Data<GqlDynamicSchema>,
-    context: Data<RwLock<GqlContext>>,
+    schema: Data<DynamicSchema>,
+    context: Data<RwLock<DataloaderContext>>,
 ) -> Result<HttpResponse, Error> {
     let context = context
         .read()
         .map_err(|_| actix_web::error::ErrorInternalServerError("Failed to read context"))?;
-    let ctx = GqlContext {
-        subgraph_name_lookup: context.subgraph_name_lookup.clone(),
-        entries_store: context.entries_store.clone(),
-        // entries_broadcaster: context.entries_broadcaster.clone(),
-    };
-    let config = ConnectionConfig::new(ctx);
+    let config = ConnectionConfig::new(context);
     let config = config.with_keep_alive_interval(Duration::from_secs(15));
-    subscriptions::ws_handler(req, stream, schema.into_inner(), config).await
+    unimplemented!()
```

Comment:
Is this unimplemented intentional?

Reply:
Yes, we'll need to get back to subscriptions!